site/cypress/e2e/Cypress-UI/profile.spec.js

```diff
@@ -85,11 +85,52 @@ describe('Test cases revolving around user profile page', () => {
 
     // Selenium test_time_zone_selection
     it('Time zone selector should work', () => {
```

Comment:
None of the tests are actually being run because they are inside of another 'it' statement, my suggestion would be to remove all of the inside 'it' statements, and have one big check for the time zone selector, and you can separate parts by comments so code readability is easier.

Reply:
I'll fix this by Tomorrow, thanks for letting me know.